src/libraries/Microsoft.PowerFx.Core/Public/Values/TableValue.cs

```diff
@@ -99,30 +99,72 @@ public virtual int Count()
         /// <summary>
         /// Lookup the record at the given 1-based index, or return an error value if out of range.
         /// </summary>
-        /// <param name="index1">1-based index.</param>
+        /// <param name="index1">1-based index.</param>
+        /// <param name="mutationCopy">copies the element, and the table entry pointing to it, when in a mutation context.</param>
         /// <returns>The record or an errorValue.</returns>
-        public DValue<RecordValue> Index(int index1)
+        public DValue<RecordValue> Index(int index1, bool mutationCopy = false)
```

Comment:
> Index(int index1, bool mutationCopy = false) [](http://example.com/codeflow?start=34&length=45)

Nit: Can you add this an overload? 

In C#, adding parameters (even with default values) is a *binary* breaking change.   That's what the Public API changes are alerting about.

Reply:
Added as overload.